sections/02_intro.md

```diff
@@ -1,54 +1,52 @@
-## Introduction
+## Introduction to Deep Learning
 
 Biology and medicine are rapidly becoming data-intensive. A recent comparison of
 genomics with social media, online videos, and other data-intensive
-disciplines suggested that genomics alone would equal or surpass other fields in
+disciplines suggests that genomics alone will equal or surpass other fields in
 data generation and analysis within the next decade
 [@doi:10.1371/journal.pbio.1002195]. The volume and complexity of these data
-present not only new opportunities, but also new challenges. Automated
-algorithms will be crucial in extracting meaningful patterns and actionable
-knowledge that allow us to better treat, categorize, or study disease, all
-within data-constrained and privacy-critical environments.
+present new opportunities, but also pose new challenges. Automated
+algorithms that extract meaningful patterns could lead to actionable
+knowledge and change how we develop treatments, categorize patients, or study diseases, all
+within privacy-critical environments.
 
-Over the past five years, a class of machine learning algorithms known as deep
-learning has revolutionized image classification and speech recognition due to
-its flexibility and high accuracy [@doi:10.1038/nature14539]. More recently,
-these algorithms have shown equally promising results in fields as diverse as
+The term deep learning has come to refer to a collection of new techniques that,
+together, have demonstrated breakthrough gains over existing best-in-class
+machine learning algorithms across several fields. For example,
+over the past five years these methods have
+revolutionized image classification and speech recognition due to
+their flexibility and high accuracy [@doi:10.1038/nature14539]. More recently,
+deep learning algorithms have shown promise in fields as diverse as
 high-energy physics [@doi:10.1038/ncomms5308], dermatology
 [@doi:10.1038/nature21056], and translation among written languages
 [@arxiv:1609.08144]. Across fields, "off-the-shelf" implementations of these
 algorithms have produced comparable or higher accuracy than previous
 best-in-class methods that required years of extensive customization, and
 specialized implementations are now being used at industrial scales.
 
-Deep learning algorithms can also be used in an exploratory, "unsupervised"
+These methods are powered by improvements to artificial
+neural networks, an idea that was first proposed in 1943
+[@doi:10.1007/BF02478259] as a model for how our brains process
+information. Since then, interest in neural networks as computational models has
+waxed and waned over time. This history is interesting in its own right
+[@doi:10.1103/RevModPhys.34.135]. In recent years, attention has shifted back to
+neural networks as hardware improvements have allowed the techniques that have come to be known as deep learning to
```

Comment:
I think the intention of "techniques that have come to be known as deep learning" it to convey that is it not a single technique but rather a multitude.  The phrase is long though.  This might be best pushed to the TODO to shorten the phrasing here.